upf/upf_cli.c

```diff
@@ -272,6 +269,10 @@ upf_nat_pool_add_del_command_fn (vlib_main_t * vm,
 	;
       else if (unformat (line_input, "block_size %u", &port_block_size))
 	;
+      else if (unformat (line_input, "min_port %u", &min_port))
```

Comment:
Together with initializing `min_port` and `max_port` to 0, this will cause `nat_pool->max_port` to be `0` if it is not specified, right? That doesn't sound correct

Reply:
Yep
so, with keeping assignment like you did, but with addition of this code lines?

Reply:
I'd also change underscores to dashes in the command (e.g. `min_port` to `min-port`) for the sake of consistency (we have breaking changes in NAT setup commands in any case, right?)
And add a check for `min_port <= max_port`, do you agree with that?

Reply:
`min_port <= max_port` should be guaranteed by assignment in upf.c with min/max constants
extra check is redundant
dashes or underscore - let's make a separate PR for this